### PR TITLE
Hero section redesign

### PR DESCRIPTION
**Pull request by jaynaxissa**
## Context

https://ywgc.atlassian.net/browse/YWGC-37?atlOrigin=eyJpIjoiN2I5OTBhYjJmMzZiNDM0MmFiMDVmYTg0NzVhNzQ2MjkiLCJwIjoiaiJ9

## Work done
- Change to a new design; wrote the web design version and the mobile design version
- Add new hero picture
- Change font size in Baseheading component to fit the new design
- Change the green button
- Add contact number at the top right corner of the home page

## Manual testing instructions

- [ ] The design should be responsive to screen size
- [ ] The web version and mobile version is a different design as for the web the picture will be on the left and for mobile it would be in the background of the text
- [ ] When "Our Services" button is clicked it will redirect to the service section of the page. And it should be smaller when it comes to a different screen size
- [ ] Alt text for the hero image in web version should show
- [ ] Font should switch to a smaller font when the screen size is smaller

## Gotchas/What I learned

- Communication is key! I find it really helpful talking to our designer to figure out some part of the design.
- Use React hooks such as useEffect and useState alongside with EventListener and ternary operator to make the screen able to change from one design to another when it's mobile or web design.

### CODE REVIEW

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -1,22 +1,107 @@
...
+  const breakpoint = 620
+  React.useEffect(() => {
+    /* Inside of a "useEffect" hook add an event listener that updates
+       the "width" state variable when the window size changes */
+    window.addEventListener('resize', () => setWidth(window.innerWidth))
+    /* passing an empty array as the dependencies of the effect will cause this
+      effect to only run when the component mounts, and not each time it updates.
+       We only want the listener to be added once */
+  }, [])
+  return width > breakpoint ? renderWeb() : renderMobile()
```
Comment (novellac) [inline src/pages/Home.jsx:104]: observation(nb): This shouldn't be necessary, we should be able to use Tailwind's `sm`, `md`, etc prefixes for styling.

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -1,22 +1,107 @@
...
+          </p>
+        </div>
+      <div className="p-6 bg-white bg-[url('/public/images/hero.png')] bg-no-repeat bg-cover">
+        <div className='flex flex-col items-center justify-center p-6 bg-white bg-opacity-80 rounded-2xl'>
+          <div className='py-5 px-16 gap-2.5'>
+            <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
+            <hr className='w-[211px] h-px mx-auto mt-4 bg-black border-0 rounded'></hr>
+          </div>
+          <div>
+            <div className='w-[358px] h-[80px] px-8'>
```
Comment (novellac) [inline src/pages/Home.jsx:66]: nitpick: Please don't use the custom pixel sizes. Instead, please discuss with the designer how we can stick to TW's default sizing.

Comment (jaynaxissa) [reply src/pages/Home.jsx]: Got it. I just messaged the designer about this. Currently waiting for the TW default sizing.

**Review by alliequintano — APPROVED** [review]
Looks good from the PM perspective 👍 Tested for accessibility and responsiveness.

**Pull request merged**